src/test/java/PalindromeCheckerTest.java

```diff
@@ -5,6 +5,7 @@
 
 public class PalindromeCheckerTest {
 
+    //visibility
```

Comment:
Think this should be private - that should be your goto visibility modifier for fields on a class.